background.ts

```diff
@@ -769,6 +773,12 @@ class BackgroundCommandWorker implements CommandWorkerInterface {
     return jsonStringifyWithWhiteSpace(cfg);
   }
 
+  factoryReset(keepSystemImages: boolean) {
+    doFactoryReset(keepSystemImages).catch((err) => {
+      console.error(err);
```

Comment:
Hmm, so if there's an error, we return success to the caller (HTTP API), and log the error.
But I don't think we can do very much about it (because we tore down the server).

Reply:
If an exception is thrown before the server is shutdown, here is where it will be displayed.